src/ui/components/VPNControllerView.qml

```diff
@@ -274,11 +294,7 @@ Item {
 
             PropertyChanges {
                 target: logoSubtitle
-                //% "Secure and private"
-                //: This refers to the user’s internet connection.
-                text: qsTrId("vpn.controller.active") + "  •  " + formatTime(VPNController.time)
-                color: "#FFFFFF"
-                opacity: 0.8
+                visible: false
             }
 
```

Comment:
My bad - I think I told you we could get rid of this little block but it looks like like we do actually need it. After that, everything is good to go. Thank you so much for all of the iterations with this!

```            
PropertyChanges {
    target: logoSubtitleOn
    visible: true
}

Reply:
I added the block back in. I didn‘t notice any difference in visual behaviour of the subtitle — were there any other side effects?

Reply:
Yeah I noticed that if I disconnected when the 'Unstable' message was visible, then reconnected, `logoSubtitleOn` was still hidden.